web/src/main/java/org/cbioportal/web/error/GlobalExceptionHandler.java

```diff
@@ -1,8 +1,6 @@
 package org.cbioportal.web.error;
 
-import org.cbioportal.service.exception.PatientNotFoundException;
-import org.cbioportal.service.exception.SampleNotFoundException;
-import org.cbioportal.service.exception.StudyNotFoundException;
+import org.cbioportal.service.exception.*;
```

Comment:
this change is against google java style ... although I understand the preference. I think our style should allow wildcards for standard java packages like java.util.*, but for our own packages I think explicit listings are better.
We can discuss this as part of the development guidelines/practices document.